tools/syz-declextract/testdata/functions.c

```diff
@@ -11,15 +11,48 @@ static void func_bar() {
 	func_foo();
 }
 
-void func_baz(int f) {
+int alloc_fd() {
```

Comment:
How about adding __fget_light() or any other function to test the flow to function arguments?

Reply:
Done.
Worked as intended.